docs/usage.md

```diff
@@ -74,7 +74,14 @@ The tool offers also the ability to flash SSL certificates to a module:
 ./arduino-fwuploader certificates flash -b arduino:samd:nano_33_iot" -a COM10 -u arduino.cc:443 -u google.com:443
 ```
 
-or you can specify a path to a file with `-f` instead of the URL of the certificate
+or you can specify a path to a file in `.der` format with `-f` instead of the URL of the certificate
+
+Due to a limitation on the handling of certs in the Nina modules, when `certificates flash` command is launched, all the
```

Comment:
```suggestion
Due to a limitation on the handling of certs in the NINA modules, when `certificates flash` command is launched, all the
```
I think this "NINA module" term is a bit obscure and potentially confusing, so I think it is best to be as consistent as possible in how it is used.